.github/workflows/main.yml

```diff
@@ -167,6 +167,6 @@ jobs:
       - name: Run Functional/Integration Tests (via hardware)
         run: |
           source devel/setup.bash
-          rostest inertial_sense_ros test_ros_bridge.launch
-          catkin_test_results || (rostest --text inertial_sense_ros test_ros_bridge.launch && catkin_test_results && false)
-
+          roscore &
```

Comment:
I don't know that this is safe... rostest is supposed to launch the roscore, and then terminate it when its finished.  Was there a specific reason for doing this way?

Reply:
Whoever wrote the test above this did it that way so when I was adjusting the calls below I just matched what was above. I know that the way of calling the tests in the upper test was ROS more normal so I matched the other pieces as well.

Reply:
Also, if you do it the "new" method then the output is more verbose. That was one big benefit.

Reply:
How do you ensure the roscore is stopped at the end correctly?   I guess I'm curious what the point of `rostest` is, if the "more ROS way" to run tests is to not use rostest?

Reply:
I understand it as catkin_make run_test_x plus catkin_test_results is the way ROS handles gtest, etc better.